group_vars/geniza/vars.yml

```diff
@@ -123,6 +123,9 @@ solr_configset: geniza
 deploy_contexts:
   - "Python unit tests"
 
+# run cron jobs only on the first machine in the host group
+cronjob_host: "{{ groups['geniza_prod'][0] }}"
```

Comment:
Comment above should have been on this line . . . where `cronjob_host` is defined. This uses runtime order, which may change from playbook run to playbook run.

Reply:
Hm, no, I think I was wrong. This is a pattern, so it does use inventory order. I think it's fine, as long as you don't add hosts to the group before the host you want to use as the `cronjob_host`.